lib/capistrano/tasks/magento.rake

```diff
@@ -448,6 +448,29 @@ namespace :magento do
         end
       end
     end
+
+    namespace :advanced-bundling do
+      desc 'Deploys advanced bundling'
+      task :deploy do
+        on release_roles :all do
+          deploy_themes = fetch(:magento_deploy_themes)
+          deploy_languages = fetch(:magento_deploy_languages)
+
+          within release_path do
+            if (File.exists?('build.js'))
```

Comment:
Use `if test "[[ -f build.js ]]"` File.exists will check if it exists on the local machine, not the remote.

Reply:
Deploys are running on the remote server? But I'll change it for the sake of consistency.

Reply:
They are, but at the same time they aren't. The rakefile gets executed locally/on the deploy server. If you use basic Ruby functions they will be executed on the system running capistrano instead of over the SSH connection